src/services/goals.js

```diff
@@ -359,9 +365,8 @@ export function reduceObjectives(newObjectives, currentObjectives = []) {
       ids: [id],
       // Make sure we pass back a list of recipient ids for subsequent saves.
       recipientIds: [objective.getDataValue('otherEntityId')],
-      ttaProvided,
+      // ttaProvided
```

Comment:
Should this be commented out?

Reply:
I removed the commented out line (this method is called in places where there will be no TTA provided or we don't care about that)